lib/line_wrapper.js

```diff
@@ -85,10 +85,10 @@ class LineWrapper extends EventEmitter {
   }
 
   wordWidth(word) {
-    return (
+    return PDFNumber(
```

Comment:
Should not be rounded only when storing the number to the pdf file?

wordWidth is called in many intermediate steps.

Reply:
we need to ensure that during the line_wrapper computations we are keeping a float32 number so we could either wrap all instances of wordWidth with the rounder, or just do it here